Formula/libiscsi.rb

```diff
@@ -12,23 +12,36 @@ class Libiscsi < Formula
     sha256 "c89e40197b9aebd712c67d43fbe7a4e085ddb7a1b58861c6e98444271fd9e383" => :yosemite
   end
 
-  option "with-noinst", "Install the noinst binaries (e.g. iscsi-test-cu)"
+  option "with-noinst", "Install the noinst binaries (examples, tests)"
+  option "with-cunit", "Install iscsi-test-cu, the iSCSI target test suite"
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
-  depends_on "cunit" if build.with? "noinst"
-  depends_on "popt"
+  depends_on "cunit" => :optional
 
   def install
+    if build.without? "cunit"
+      inreplace "test-tool/Makefile.am", "bin_PROGRAMS =", "noinst_PROGRAMS ="
+    end
     if build.with? "noinst"
       # Install the noinst binaries
-      inreplace "Makefile.am", "noinst_PROGRAMS +=", "bin_PROGRAMS +="
+      inreplace ["tests/Makefile.am", "examples/Makefile.am"], "noinst_PROGRAMS =", "bin_PROGRAMS ="
     end
```

Comment:
I think we should just default to the options that don't require `inreplace` patching and remove the others. If upstream wants to provide these as options they should do so as `configure` options.